F1WM/Mapping/NewsMappingProfile.cs

```diff
@@ -10,7 +10,7 @@ public class NewsMappingProfile : Profile
 		public NewsMappingProfile()
 		{
 			CreateMap<News, NewsSummary>()
-				.ForMember(api => api.TopicIcon, o => o.MapFrom(db => db.Topic.TopicIcon));
+				.ForMember(api => api.TopicIcon, o => o.MapFrom(db => db.Topic.Icon));
```

Comment:
This refactor effectively makes this line redundant. Automapper can deduce the mapping by itself.